bin/pkg/pkg.ml

```diff
@@ -1,22 +1,23 @@
 open Import
 
-let info =
-  let doc = "Experimental package management" in
-  let man =
-    [ `S "DESCRIPTION"
-    ; `P {|Commands for doing package management with dune|}
-    ; `Blocks Common.help_secs
-    ]
-  in
-  Cmd.info "pkg" ~doc ~man
+let man =
+  [ `S "DESCRIPTION"
+  ; `P {|Commands for doing package management with dune|}
```

Comment:
Maybe "Commands for OCaml package management with Dune"

Reply:
I just extract the code, but I can modify that :wink: